gemfiles/Gemfile-sorbet-minimum

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+gem("sorbet-static-and-runtime", "0.5.9204")
```

Comment:
As noted in my description, this matches the minimum version we specify in `tapioca.gemspec` (`">= 0.5.9204"`).

1. **This version [does not exist](https://rubygems.org/gems/sorbet-static-and-runtime/versions).**
  _We should either switch to a version number that does exist, or fallback to specifying Sorbet dependencies individually._
1. **Ideally we'd do this dynamically so this file doesn't need to be kept in sync.**
  _Perhaps simplest to just do string matching on the `Gemfile` and extract the version?_

Reply:
For 1. I think we should first create a PR that bumps the version to an existing one. The first version for `sorbet-static-and-runtime` is [v0.5.9519](https://rubygems.org/gems/sorbet-static-and-runtime/versions/0.5.9519) which is almost a year old. We can bump to something more recent.

Reply:
Let's keep the one you used in #1260 for the time being 👍 

Thanks!

Reply:
To tie up loose ends, #1260 ended up bumping us to the first version that allows us to remove our feature flag checks (because all features are supported).

```ruby
spec.add_dependency("sorbet-static-and-runtime", ">= 0.5.9892")
```

Reply:
Okay, I had to nudge it forward again to `0.5.9896`, because `0.5.9892` doesn't actually exist either 😅